src/App.tsx

```diff
@@ -1,14 +1,60 @@
 /* eslint-disable max-len */
-import React from 'react';
+import { useState, useEffect } from 'react';
+
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [todosFromServer, setTodosFromServer] = useState<Todo[]>([]);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [todoId, setTodoId] = useState(0);
+  const [selectedTodo, setSelectedTodo] = useState<Todo>();
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
it isn't correct, when you mount component isLoading will be true, but isLoading should be set at the start and end of the asynchronous action

example:

```
  const fetchData = () => {
    setLoading(true)
    fetch().then(() => {...}).finally(() => {
      setIsLoading(false)
    })
  }
```